deploy/operator/capi/deploy_capi_cluster.sh

```diff
@@ -60,6 +60,13 @@ fi
 
 if [ "${DISCONNECTED}" = "true" ]; then
     ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE="${LOCAL_REGISTRY}/$(get_image_without_registry ${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE})"
+
+    oc get namespace hypershift || oc create namespace hypershift
+    oc get secret "${ASSISTED_PULLSECRET_NAME}" -n hypershift || \
+      oc create secret generic "${ASSISTED_PULLSECRET_NAME}" --from-file=.dockerconfigjson="${ASSISTED_PULLSECRET_JSON}" --type=kubernetes.io/dockerconfigjson -n hypershift
```

Comment:
I see code below that's similar, but using a different namespace.  Which is correct?

The code below:
```
oc get namespace "${SPOKE_NAMESPACE}" || oc create namespace "${SPOKE_NAMESPACE}"
oc get secret "${ASSISTED_PULLSECRET_NAME}" -n "${SPOKE_NAMESPACE}" || \
    oc create secret generic "${ASSISTED_PULLSECRET_NAME}" --from-file=.dockerconfigjson="${ASSISTED_PULLSECRET_JSON}" --type=kubernetes.io/dockerconfigjson -n "${SPOKE_NAMESPACE}"
```

Reply:
Thanks for taking a look Avishay! This is similar since we need the same pull secret created in the `hypershift` namespace for the hypershift operator pods